create-i18n/index.d.ts

```diff
@@ -82,6 +82,7 @@ export function createI18n<Locale extends string, BaseLocale extends Locale>(
   locale: LocaleStore<Locale>,
   opts: {
     baseLocale?: BaseLocale
+    initialCache?: Record<Locale, Translations>
```

Comment:
Let’s rename to `cache` (I prefer shorter names in nano-libs).

And please update Size Limi config.

Reply:
Done